pyuvsim/utils.py

```diff
@@ -141,7 +149,15 @@ def zenithangle_azimuth_to_altaz(zenith_angle, azimuth):
 
 
 def strip_extension(filepath, ext=None):
-    """ Remove extension from file. """
+    """
+    Remove extension from file.
+
+    Parameters
+    ----------
+    ext : str
+        Extenstion to remove. If not specified, only 'uvfits', 'uvh5', 'yaml' extensions
+        are removed.
+    """
     if '.' not in filepath:
```

Comment:
this function seems brittle -- thought of using the `pathlib.Path` module for this kind of stuff?

Reply:
Looks like this is only called in this module in the `check_file_exists_and_increment` function. My first reaction is I agree and I'd favor using this instead:
https://docs.python.org/3/library/os.path.html#os.path.splitext

But then I started worrying about what HERA does where it adds lots of "extensions" to filenames and I wonder if that was the concern.

@aelanman can you comment?

Reply:
I'll make an issue to follow up on this later.